ontology/yaml/resources/LIGHTING/entity_types/LT.yaml

```diff
@@ -37,3 +37,20 @@ LT_DC_SC:
   - LT
   - DC 
   - SC
+
+#Noncanonicl types for Delmatic lighting controller at AU-SYD-ODI
+LGRP_DC_MS_IM:
+  description: "Delamtic controller lighting groups with brightness, motion sensing and illuminance monitoring"
+  is_canonical: false
+  implements:
+  - LGRP
+  - DC
+  - MS
+  - IM
+
+ELT_DELMATIC:
```

Comment:
Remove; we do not want references to specific manufacturers, as this is generally unhelpful -- the fields and functionalities of the entities are sufficient to describe them.

Reply:
Changed to ELT_GENERIC